### ISSUE TITLE
Document usage of phimpme app with standard Drupal

### ISSUE THREAD

**Issue by mariobehling** [fossasia/phimpme-drupal#1]
The Drupal version and modules in this repository are outdated. The goal is to use the phimpme app as much as possible with a standard vanilla Drupal. Therefore, please:
- [ ] Install a standard Drupal from http://drupal.org and test or implement the app with the standard install
- [ ] Document how to get the app working with Drupal in the Readme.md
- [ ] Delete any files/folders no longer required in this repository

**Comment by arpitx165** [comment]
@mariobehling  Hi,Can i work on this issue ? I have already worked on Drupal 7.

**Comment by mariobehling** [comment]
@arpitx165 Yes, please work on this issue, but you need to join the org first, before we can assign the issue to you. I have sent you an invite.

**Comment by arpitx165** [comment]
@mariobehling Is there anything need to read before working on this issue ?

**Comment by pa1pal** [comment]
Hi @arpitx165 . This is a sample drupal blog to test phimpme android app. [https://github.com/fossasia/phimpme-android](https://github.com/fossasia/phimpme-android)
The objective of the app is to upload images to many networks at once. One of them is drupal. The app is quite old so most of the things are not working now. I tested drupal on app there is some problem with parsing.

**Comment by arpitx165** [comment]
@pa1pal  Link is not working

**Comment by pa1pal** [comment]
now working.
